modules/custom/social_gdpr/social_gdpr.install

```diff
@@ -52,7 +52,7 @@ function social_gdpr_update_8902() {
  * Permissions to check:
  * - "translate data_policy".
  */
-function social_gdpr_update_111101(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.